libs/x-content/src/main/java/org/elasticsearch/xcontent/XContentBuilder.java

```diff
@@ -504,6 +504,22 @@ private XContentBuilder values(float[] values) throws IOException {
         return this;
     }
 
+    public XContentBuilder array(String name, byte[] values) throws IOException {
```

Comment:
nit: I am seeing that the original author meant to keep the methods for the same type grouped together. Each block starts with a comment. These two nee methods are in the middle of the float block, maybe they deserve their own block just for consistency.

Reply:
I will move up to the byte commented section :)